core/src/main/java/org/infinispan/persistence/manager/PersistenceManagerImpl.java

```diff
@@ -172,7 +173,9 @@ public void start() {
 
             // Now schedule the availability check
             long interval = configuration.persistence().availabilityInterval();
-            availabilityFuture = persistenceExecutor.scheduleAtFixedRate(this::pollStoreAvailability, interval, interval, TimeUnit.MILLISECONDS);
+            if (interval > 0) {
+               availabilityFuture = persistenceExecutor.scheduleAtFixedRate(this::pollStoreAvailability, interval, interval, TimeUnit.MILLISECONDS);
```

Comment:
If Wildfly could expose `availabilityFuture` as a parameter, we could use `-1` to disable it if needed.

Reply:
For the infinispan server I have created a PR to expose the availability parameters https://github.com/infinispan/infinispan/pull/6503. However, the infinispan subsystem provided as part of wildfly 14 is out of our control.

What we recommend is to utilise our `wildfly-modules` zip to utilise the latest version of Infinispan  on Wildfly. This will allow you to use our subsystem extensions which have more configuration options. Please read our [user guide](http://infinispan.org/docs/stable/user_guide/user_guide.html#infinispan_modules_for_wildfly_eap) for more details.

Reply:
Cool, thanks a lot for the info. 👍 😄